### PR TITLE
AQM Rebuild

### PR DESCRIPTION
**Pull request by AlecIsaacson**
** Preview Only **

This is a rebuild of the AQM content to reflect the new alert architecture and the NrAiIncident event type.

** Preview Only **

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Comment by a-james-faria** [comment]
@AlecIsaacson - two old KPIs `% ack` and `MTTI` were left in the definitions.

File: src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide.mdx [hunk 1]
```diff
@@ -79,20 +74,20 @@ Incident count is the number of incidents generated over a period of time. Typic
 **Best practices:**
 * Ensure condition settings are intended to detect real business impact.
 * Ensure condition settings are detecting abnormal behavior.
-* Communicate that the incident details "Acknowledge" feature helps measure meaningful and actionable alerts. See [Percentage incident acknowledge KPI](#kpi-user-engagement).
+* Communicate that the incident details "Acknowledge" feature helps measure meaningful and actionable alerts. See [percentage incident acknowledge KPI](#kpi-user-engagement).
```
Comment (a-james-faria) [inline src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide.mdx:77]: Looks like you tried to delete this but it was reinserted.

File: src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide.mdx [hunk 2]
```diff
@@ -129,18 +124,20 @@ Percentage of incidents where the duration of the incident is under five minutes
 
 ### User engagement [#kpi-user-engagement]
 
-You should measure the value of an incident by the amount of attention it receives. Engagement in this context is measured by whether or not an incident has been acknowledged.
+You should measure the value of an incident by the amount of attention it receives. Here, we measure engagement by whether or not an incident has been acknowledged.
```
Comment (a-james-faria) [inline src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide.mdx:127]: old kpi

File: src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide.mdx [hunk 3]
```diff
@@ -129,18 +124,20 @@ Percentage of incidents where the duration of the incident is under five minutes
...
+There is a significant difference between measuring the moment of incident awareness vs. acknowledging the moment resolution activity begins. If you are using an integration with New Relic alerts, be sure that the "acknowledge" event that is sent to New Relic is triggered when resolution activity begins, not when the incident is sent to the external incident management tool. 
+
+For more information regarding standard incident management processes, see "[Incident management process: 5 steps to effective resolution, posted on August 31, 2020 by OnPage Corporation.](https://www.onpage.com/incident-management-process-5-steps-to-effective-resolution) in reference to [ITIL4](https://itsm.tools/its-here-itil-4-explained)"
 
 <CollapserGroup>
   <Collapser
     id="kpi-pct-ack"
     title="Percentage acknowledged KPI"
```
Comment (a-james-faria) [inline src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide.mdx:138]: old kpi

File: src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide.mdx [hunk 4]
```diff
@@ -129,18 +124,20 @@ Percentage of incidents where the duration of the incident is under five minutes
...
   >
-Incidents acknowledged identifies the percentage of incidents that have been engaged with and had their acknowledged property set to true. Typically you should compare the current and previous weeks.
+Incidents acknowledged identifies the percentage of incidents that have been engaged with by having their acknowledgement flag set to true. Typically you should compare the current and previous weeks.
```
Comment (a-james-faria) [inline src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide.mdx:140]: looks like it was reinserted

File: src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide.mdx [hunk 5]
```diff
@@ -154,7 +151,7 @@ Incidents acknowledged identifies the percentage of incidents that have been eng
     id="kpi-mtti"
     title="Mean time to investigate (MTTI) KPI"
   >
-Mean time to investigate identifies the average time it takes for an incident to be triaged. Typically you should compare the current and previous weeks.
+Mean time to investigate identifies the average time it takes for an incident to be acknowledged. Typically you should compare the current and previous weeks.
```
Comment (a-james-faria) [inline src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide.mdx:154]: looks like it was reinserted

File: src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide.mdx [hunk 6]
```diff
@@ -171,106 +168,103 @@ Before you begin, if you don't have equivalent experience, complete the [New Rel
...
+1. Are the alerts telling us something about a resource that needs to be fixed?  If so, then fix the problem and see if the alert volume decreases.
+1. Are the alerts telling us about something that actually requires an immediate response?  If not, then adjust or disable the policy.
+1. Are the policy thresholds set properly?  If not, then consider adjusting the thresholds.
 
-The most critical portion of this task is educating your team on the importance of acknowledging incident alerts, because that's how the alert's value is determined. In general, instruct them to follow these guidelines:
+One of the most significant parts of this task is educating your team on the importance of acknowledging incident alerts, because that's how an alert's value is determined. In general, instruct them to follow these guidelines:
 * If you look at an alert and decide to take any sort of further investigative action, acknowledge the alert.
 * If you typically close an alert without doing anything else, do not acknowledge the alert.
 * If the incident alert is always on, do not close or acknowledge it. For further details, see [Second enablement session](#perform-enablement-two).
```
Comment (a-james-faria) [inline src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide.mdx:222]: old kpi verbiage

File: src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide.mdx [hunk 7]
```diff
@@ -171,106 +168,103 @@ Before you begin, if you don't have equivalent experience, complete the [New Rel
...
-You can use the [first session template presentation](https://docs.google.com/presentation/d/1X1HHEfDx30XpaB65bgn_OCLB-lG72zuRKQ8HnW3GJw4/edit?usp=sharing) to communicate this material to your stakeholders.
 
-### Accumulate AQM data [#accumulate-data]
+### Accumulate AQM alert engagement data [#accumulate-data]
 
-The overall process requires at least two weeks of data before it can proceed. During this time, you should periodically check the following items:
-* Confirm that incident alert event data is accumulating.
-* Confirm that the webhook is attached to every alert policy.
-* Ensure that incident responders are following the alert acknowledgement guidelines.
+The process of measuring alert engagement requires at least two weeks of data before it can proceed. During this time, you should periodically check to see if incident responders are following the alert acknowledgement guidelines outlined in the the [first enablement session](#perform-enablement-one).
```
Comment (a-james-faria) [inline src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide.mdx:227]: old kpi verbiage

File: src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide.mdx [hunk 8]
```diff
@@ -154,7 +151,7 @@ Incidents acknowledged identifies the percentage of incidents that have been eng
     id="kpi-mtti"
     title="Mean time to investigate (MTTI) KPI"
```
Comment (a-james-faria) [inline src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide.mdx:152]: old kpi

File: src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide.mdx [hunk 9]
```diff
@@ -55,17 +50,17 @@ You'll use the AQM process to collect and measure the following KPIs:
 * Mean time to close (MTTC)
 * Percent under 5 minutes
 
-[**User engagement**](#kpi-user-engagement), which includes: 
+[**Incident engagement**](#kpi-incident-engagement), which includes: 
 * Mean time to investigate (MTTI)
 * % of incidents investigated
```
Comment (a-james-faria) [inline src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide.mdx:55]: old kpi

File: src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide.mdx [hunk 10]
```diff
@@ -79,20 +74,20 @@ Incident count is the number of incidents generated over a period of time. Typic
...
-Accumulated incident duration is the total sum of minutes that all the incidents accumulated over a period of time. Typically you should compare the current and previous weeks.
+Accumulated incident duration is the total sum of minutes that all incidents have accumulated over a period of time. Typically you should compare the current and previous weeks.
 
 **Goal:** Reduce the total accumulated minutes of incidents.
 
 **Best practices:**
-* Do not manually close incidents. Manual closure will skew the real duration of incident length.
+* Do not manually close incidents that do not motivate some sort of investigatory action. Manual closure will skew the real duration of incident length.
 * Eliminate alerts that do not result in any remediation actions from the recipients.
 * Improve percent investigated and mean-time-to-investigate KPIs by communicating their importance in improving detection and response times.
```
Comment (a-james-faria) [inline src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide.mdx:92]: old kpi reference

File: src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide.mdx [hunk 11]
```diff
@@ -171,106 +168,103 @@ Before you begin, if you don't have equivalent experience, complete the [New Rel
...
+1. Are the alerts telling us something about a resource that needs to be fixed?  If so, then fix the problem and see if the alert volume decreases.
+1. Are the alerts telling us about something that actually requires an immediate response?  If not, then adjust or disable the policy.
+1. Are the policy thresholds set properly?  If not, then consider adjusting the thresholds.
 
-The most critical portion of this task is educating your team on the importance of acknowledging incident alerts, because that's how the alert's value is determined. In general, instruct them to follow these guidelines:
+One of the most significant parts of this task is educating your team on the importance of acknowledging incident alerts, because that's how an alert's value is determined. In general, instruct them to follow these guidelines:
 * If you look at an alert and decide to take any sort of further investigative action, acknowledge the alert.
```
Comment (a-james-faria) [inline src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide.mdx:220]: old kpi reference

**Review by a-james-faria — CHANGES_REQUESTED** [review]
Look like some old KPI text was reinserted and some other text was missed.

**Review by homelessbirds — APPROVED** [review]

**Pull request merged**